csharp/test/Microsoft.ML.OnnxRuntime.Tests.Android/RunAllTest.cs

```diff
@@ -0,0 +1,107 @@
+﻿using OpenQA.Selenium.Appium;
+using OpenQA.Selenium;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.ML.OnnxRuntime.Tests.Android
+{
+    [TestFixture]
+    public class RunAllTest : BrowserStackTest
+    {
+        public AppiumElement FindAppiumElement(String xpathQuery, String text)
+        {
+            IReadOnlyCollection<AppiumElement> appiumElements = driver.FindElements(By.XPath(xpathQuery));
+
+            foreach (var element in appiumElements)
+            {
+                if (element.Text.Contains(text))
+                {
+                    return element;
+                }
+            }
+            // was unable to find given element
+            throw new Exception(String.Format("Could not find {0}: {1} on the page.", xpathQuery, text));
+        }
+
+        public AppiumElement FindAppiumElementThenClick(String xpathQuery, String text)
+        {
+            AppiumElement appiumElement = FindAppiumElement(xpathQuery, text);
+            appiumElement.Click();
+            return appiumElement;
+        }
+
+        public (int, int) GetPassFailCount()
+        {
+            int numPassed = -1;
+            int numFailed = -1;
+
+            IReadOnlyCollection<AppiumElement> labelElements = driver.FindElements(By.XPath("//android.widget.TextView"));
+
+            for (int i = 0; i < labelElements.Count; i++)
+            {
+                AppiumElement element = labelElements.ElementAt(i);
+
+                if (element.Text.Equals("✔"))
+                {
+                    i++;
+                    numPassed = int.Parse(labelElements.ElementAt(i).Text);
+                }
+
+                if (element.Text.Equals("⛔"))
+                {
+                    i++;
+                    numFailed = int.Parse(labelElements.ElementAt(i).Text);
+                    break;
+                }
+            }
+
+            Assert.That(numPassed, Is.GreaterThanOrEqualTo(0), "Could not find number passed label.");
+            Assert.That(numFailed, Is.GreaterThanOrEqualTo(0), "Could not find number failed label.");
+
+            return (numPassed, numFailed);
+        }
+
+        [Test]
+        public async Task ClickRunAllTest()
+        {
+            AppiumElement runAllButton = FindAppiumElementThenClick("//android.widget.Button", "Run All");
```

Comment:
I don't know all the context here. what is being run by this?

Reply:
Ah sorry, added a README that should provide more context. 

This line uses Appium (service that provides UI automation) to search on the page for "//android.widget.Button" then after finding it, clicks the button.

Reply:
thanks for adding the readme